qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import division
+import warnings
+
+from os import remove
+from os.path import exists, join
+
+from natsort import natsorted
+# This is the only folder in qiita_pet that should import from outside
+# qiita_pet. The idea is this proxies the call and response dicts we expect
+# from the Qiita API once we build it. This will be removed and replaced with
+#  API calls when the API is complete.
+from pandas.parser import CParserError
+from qiita_db.metadata_template.sample_template import SampleTemplate
+from qiita_db.study import Study
+from qiita_core.util import execute_as_transaction
+
+from qiita_db.metadata_template.util import (load_template_to_dataframe,
+                                             looks_like_qiime_mapping_file)
+from qiita_db.util import get_mountpoint
+from qiita_db.exceptions import (QiitaDBColumnError, QiitaDBExecutionError,
+                                 QiitaDBDuplicateError, QiitaDBError,
+                                 QiitaDBDuplicateHeaderError)
+from qiita_ware.metadata_pipeline import (
+    create_templates_from_qiime_mapping_file)
+from qiita_ware.exceptions import QiitaWareError
+from qiita_pet.util import convert_text_html
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def sample_template_info(samp_id, user_id):
```

Comment:
Basically, IMOO sample_template_info is too general and if you search for it in the code it could return a lot of hits. However, I we use get_request or post_request on the names it might simplify this. For example, you could quickly get all the funcs that do a post_request and see which one you need to modify. I feel like this is a good idea after spending a lot of time making modifications to the code and trying to find the correct function to change and hoping to have something like this to make it easier.